src/libs/actions/IOU.js

```diff
@@ -237,6 +241,9 @@ function buildOnyxDataForMoneyRequest(
                           [iouCreatedAction.reportActionID]: {
                               errors: ErrorUtils.getMicroSecondOnyxError('iou.error.genericCreateFailureMessage'),
                           },
+                          [iouAction.reportActionID]: {
+                              errors: ErrorUtils.getMicroSecondOnyxError(null),
+                          },
```

Comment:
Can you please state where the `iouAction.reportActionID` errors field is being set?

Reply:
I don't understand, but the reason we're setting this is so we have the opacity effect for the report action in the IOU report.

Reply:
Okay, so I understand that we had no errors at all? So this is not related to remove redundant error messages but to the missing opacity itself?

Reply:
Correct. Same for report preview action.

Reply:
This one has an error if the report is not new.

Reply:
This does not look correct, can you add an error for this?

Reply:
You mean because there's no error message anywhere? It wasn't there before as well (It's one of the bullet points - https://github.com/Expensify/App/pull/20666#issuecomment-1589127887). 

Normally the error message would come with the create action, but that is hidden for iou reports, I thought maybe this is a feature request and the design team should tell us where to put the error message. If you still want me to add an error message in this action instead of the null error let me know.

Reply:
Hmm it looks we already have an error set here https://github.com/Expensify/App/blob/78145cfdc3f8449f4c42db064890b7e0fb90553b/src/libs/actions/IOU.js#L202

Reply:
Correct, that is for the report, we have one for the create action too here, but this action is hidden in the frontend.

https://github.com/Expensify/App/blob/78145cfdc3f8449f4c42db064890b7e0fb90553b/src/libs/actions/IOU.js#L238

Reply:
I see, let's not worry about this then.